core/src/sql/v1/fmt.rs

```diff
@@ -153,11 +153,11 @@ thread_local! {
 	// `thread_local!` so all accesses can use `Ordering::Relaxed`.
 
 	/// Whether pretty-printing.
-	static PRETTY: AtomicBool = AtomicBool::new(false);
+	static PRETTY: AtomicBool = const { AtomicBool::new(false) };
```

Comment:
I don't understand why this change is necessary.

Reply:
It is a clippy warning, here is the reference to the warning along with it's justification: [https://rust-lang.github.io/rust-clippy/master/index.html#/thread_local_initializer_can_be_made_const](https://rust-lang.github.io/rust-clippy/master/index.html#/thread_local_initializer_can_be_made_const)